lib/localeapp/cli/install.rb

```diff
@@ -79,13 +83,25 @@ def write_config_file
           write_rails_config
         end
 
+        def write_dotenv
+          @output.puts "NOTICE: .env detected. Your API key was saved there."
+          File.open('.env', 'a') do |file|
```

Comment:
Yes, this project coding standards prefer double quote.

No need to change existing code unless really needed (changes on same
line for example), but if you add new code, please favor double
quotes.